README.md

```diff
@@ -199,6 +201,10 @@ var pbjs = pbjs || {};
 pbjs.Promise = myCustomPromiseConstructor;
 ```
 
+#### Yield helper
+
+The `pbYield()` utility yields to `scheduler.yield()` when available, allowing long-running tasks to be broken up. Call `await pbYield()` inside async functions to cooperate with the scheduler.
+
```

Comment:
Who's the audience for this? IMO the README is directed at publishers, who have no way to call an internal utility function..

Reply:
i was thinking in reaction to this should it go in CONTRIBUTING or just get deleted and i chose just delete it